lib/internal/process/pre_execution.js

```diff
@@ -52,29 +52,19 @@ const {
 } = require('internal/v8/startup_snapshot');
 
 function prepareMainThreadExecution(expandArgv1 = false, initializeModules = true) {
-  prepareExecution({
-    expandArgv1,
-    initializeModules,
-    isMainThread: true,
-  });
+  return prepareExecution(expandArgv1, initializeModules, true);
```

Comment:
That seems unrelated, can you revert?

Reply:
It's an optimization, I mentioned in the PR description.

Reply:
It would still be helpful to revert it so to have a smaller diff to review.

Reply:
These were parameters until the number of parameters reach 3 which in general means it's time to use an option object. I don't think turning it into parameters would make much difference in terms of performance - this is only run once during startup anyway, so the function is probably only interpreted. The performance difference of property access v.s. argument is already small even in compiled functions, let alone interpreted ones.